travis/travis_install_nightly

```diff
@@ -33,11 +33,24 @@ fi
 
 : ${ODOO_REPO:="odoo/odoo"}  # default value, if not set
 IFS="/" read -a REPO <<< "${ODOO_REPO}"
-ODOO_URL="https://github.com/${REPO[0]}/${REPO[1]}/archive/${VERSION}.tar.gz"
-
-echo "Installing Odoo ${ODOO_URL}"
-wget -nv -O odoo.tar.gz ${ODOO_URL}
-tar -xf odoo.tar.gz -C ${HOME}
+export REMOTE=$(python -c "print '${REPO[0]}'.lower()")
+export REPO_NAME=$(python -c "print '${REPO[1]}'.lower()")
+export BRANCH=$(python -c "print '${VERSION}'.lower()")
+export ODOO_PATH=${HOME}/$REPO_NAME-$BRANCH
+if [ -z "${REPO_CACHED}"  ]; then
+    export ODOO_URL="https://github.com/$REMOTE/$REPO_NAME/archive/$BRANCH.tar.gz"
+    echo "Installing Odoo $ODOO_URL"
+    wget -nv -O odoo.tar.gz $ODOO_URL
+    tar -xf odoo.tar.gz -C ${HOME}
+else
+    echo "Using Odoo from cache ${REPO_CACHED}"
+    chown `(whoami)`:`(whoami)` -R ${REPO_CACHED}
```

Comment:
IMO the group should be obtained with `id -gn`

```
dlasley@runbot:~$ id -gn
dlasley
```

As seen above, primary group is chosen from the user groups:
```
dlasley@runbot:~$ groups `whoami`
dlasley : dlasley adm cdrom sudo dip plugdev lpadmin sambashare
```